inc/config.php

```diff
@@ -45,6 +45,25 @@
 Session::setPath();
 Session::start();
 
+//init cache
+try {
+   $cache_conf = GLPI_CACHE_CONFIG;
+   if (!isset($cache_conf['adapter']['options'])
+       || !isset($cache_conf['adapter']['options']['namespace'])) {
+      $cache_conf['options']['namespace'] = 'glpicache';
+   }
+   if (!defined('TU_USER') || defined('CACHED_TESTS')) {
+      $GLPI_CACHE = Zend\Cache\StorageFactory::factory($cache_conf);
+   } else {
+      $GLPI_CACHE = false;
+   }
+} catch (\Zend\ServiceManager\Exception\ServiceNotFoundException $e) {
```

Comment:
IIRC, exception name have changed in some version, probably better to catch all

Reply:
Oh... I was not aware of that. I'll change

Reply:
see https://github.com/glpi-project/glpi/commit/9474d574